packages/strapi/lib/commands/configurationRestore.js

```diff
@@ -0,0 +1,126 @@
+'use strict';
+
+const _ = require('lodash');
+const fs = require('fs');
+const strapi = require('../index');
+
+/**
+ * Will restore configurations. It reads from a file or stdin
+ * @param {string} file filepath to use as input
+ * @param {string} strategy import strategy. one of (replace, merge, keep, default: replace)
+ */
+module.exports = async function({ file: filePath, strategy = 'replace' }) {
+  const input = filePath ? fs.readFileSync(filePath) : await readStdin(process.stdin);
+
+  const app = await strapi().load();
+
+  let dataToImport;
+  try {
+    dataToImport = JSON.parse(input);
+  } catch (error) {
+    throw new Error(`Invalid input data: ${error.message}. Expected a valid JSON array.`);
+  }
+
+  if (!Array.isArray(dataToImport)) {
+    throw new Error(`Invalid input data. Expected a valid JSON array.`);
+  }
+
+  const importer = createImporter(app.db, strategy);
+
+  for (const config of dataToImport) {
+    await importer.import(config);
+  }
+
+  console.log(
+    `Successfully imported ${dataToImport.length} configuration entries using the "${strategy}" strategy.`
```

Comment:
Maybe for a v2 (but if you like the idea let's do it now):
A nice to have would be having : `Successfully imported configuration with ${strategy} strategy. Configuration entries: ${createdNb} created, ${replacedNb} replaced, ${mergedNb} merged, ${untouched} untouched.`

The user would directly see if he/she did something wrong (for instance if 0 are created).
`replaced` and `merged` are exclusive so we only need to print one of the two

Reply:
Awesome idea

Reply:
(might also be good to allow dumping message output in a json format too for automation purposes, nice to be able to pipe log output straight into another application)